frontend/netlify.toml

```diff
@@ -0,0 +1,23 @@
+[build]
```

Comment:
To confirm, will running this demo on netlify work in this state? Or should this be removed?